app/styles/custom-components/_vlc-agenda-items.scss

```diff
@@ -199,3 +192,52 @@ $i: 0;
   opacity: 1;
   color: $napier-green;
 }
+
+.vlc-agenda-items__body {
+  align-items: center;
+  padding: 1.2rem 1.8rem;
+
+  .vlc-agenda-items__body-padding {
+    padding-bottom: 1.5rem;
+  }
+}
+
+.vlc-agenda-items__remarks {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.vlc-agenda-items__icons {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  .vlc-agenda-items__icons::before {
+    color: #FFF;
+  }
```

Comment:
Does this even work? Shouldn't you use `&` when using the nested approach? You would write this if you put this block next to the one it's in. Here it looks like you're selecting a `vlc-agenda-items__icons` element within another `vlc-agenda-items__icon` container, and selecting its `::before`. 🤔